.github/workflows/pkgci.yml

```diff
@@ -108,7 +101,7 @@ jobs:
     name: Test Sharktank
     needs: [setup, build_packages]
     if: contains(fromJson(needs.setup.outputs.enabled-jobs), 'test_sharktank')
-    uses: ./.github/workflows/pkgci_test_sharktank.yml
+    uses: ./.github/workflows/pkgci_sharktank.yml
```

Comment:
Ah, and maybe keep the `pkgci_test_sharktank` file name here so it stays sorted next to other test workflows: https://github.com/iree-org/iree/tree/main/.github/workflows